lib/librte_hash/rte_hash.h

```diff
@@ -61,6 +61,11 @@ struct rte_hash_parameters {
 /** @internal A hash table structure. */
 struct rte_hash;
 
+/** @internal A hash table bucket iterator state structure. */
+struct rte_bucket_iterator_state {
```

Comment:
Replace the word `bucket` with `conflict` in the comment and in the name of the `struct`.